Example/ios/Podfile.lock

```diff
@@ -749,4 +749,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 86e380a4262db238c7a45428750af2d88465585c
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.11.3
```

Comment:
Something went wrong here? 1.11.3 is quite outdated

Reply:
To be honest I don't know why it's targetting that old version, but I see that in other examples cocoapods version is the same 🤔

Reply:
I think it comes from the version installed on your device